lib/pgn/game.rb

```diff
@@ -3,29 +3,33 @@
 module PGN
   class MoveText
     attr_accessor :notation, :annotation, :comment, :variations
-    
-    def initialize(notation, annotation = nil, comment = nil, variations = nil)
+
+    def initialize(notation, annotation = nil, comment = nil, variations = [])
       @notation = notation
       @annotation = annotation
-      @comment = comment
+      @comment = clean_text(comment)
       @variations = variations
     end
-    
+
     def ==(m)
-      self.to_s == m.to_s
+      to_s == m.to_s
     end
-    
+
     def eql?(m)
       self == m
     end
-    
+
     def hash
       @notation.hash
     end
-    
+
     def to_s
       @notation
     end
+
+    def clean_text(text)
+      text&.gsub(/{(.*)}/, '\1')&.gsub(/\s+/, ' ')&.strip
```

Comment:
Why get rid of curly braces?

Reply:
just realized I did not open a new branch for the bug fixes I proposed. My master diverged too much from the gem already. It has many fixes/features now. 

The curly braces in particular I just thought it made more sense to remove them alltogether in a later commit

Reply:
There are a couple of more additions in my master , in the case you are interested, at least: 

1. Adding the pgn and game comment string to game object
1. Fix bug when there are no moves in pgn ( only comments) 
1. Properly handle  "don't care" variations

Reply:
How Can I allow tag_values with special characters like Ññ?

for example in the line `[WhiteTeam "NARIÑO"]` I got this error:

```
Encoding::CompatibilityError:
       incompatible encoding regexp match (UTF-8 regexp with ISO-8859-1 string)
```